tutorials/compute-quickstart.md

```diff
@@ -0,0 +1,280 @@
+in the upper-right corner of the console.
+
+### Wait for the instance creation to finish
+
+The instances need to finish being created before the tutorial can proceed. To track the progress of this activity
+and others, click the [**Notifications**][spotlight-notification-menu] button in the navigation bar
+in the upper-right corner of the console.
+
+## Set up the backend instance
+
+In this section, you install a MongoDB database on the backend instance to save your data.
+
+### Connect to the instance
+
+Connect to the VM using SSH:  
+
+```bash
+gcloud compute --project "{{project-id}}" ssh --zone [backend-zone] [backend-name]
+```
+
+Replace `[backend-zone]` and `[backend-name]` with the zone and name of the instance that you created.
+
+If this is your first time using SSH from Cloud Shell, follow the instructions to create a private key.
+
+It may take several minutes for the SSH key to propagate.
+
+### Install the backend database
+
+Update packages:
+
+```bash
+sudo apt-get update
+```
+
+Install the backend MongoDB database:
+
+```bash
+sudo apt-get install mongodb
+```
+
+When asked if you want to continue, enter `Y`.
+
+### Run the database
+
+The MongoDB service starts when you install it. You must stop it so that you can change how it runs:
+
+```bash
+sudo service mongodb stop
+```
+
+Create a directory for MongoDB:
+
+```bash
+sudo mkdir $HOME/db
+```
+
+Run the MongoDB service in the background on port 80:
+
+```bash
+sudo mongod --dbpath $HOME/db --port 80 --fork --logpath /var/tmp/mongodb
+```
+
+Exit the SSH session and disconnect from the backend instance:
+
+```bash
+exit
+```
+
+## Set up the frontend instance
+
+In this section, you install the dependedncies for the web app and then install and run the frontend web app.
+
+### Connect to the instance
+
+Cnnect to the VM with SSH: Enter the zone and name of the
+instance you created.
+
+```bash
+gcloud compute --project "{{project-id}}" ssh --zone [frontend-zone] [frontend-name]
+```
+
+Replace `[frontend-zone]` and `[frontend-name]` with the zone and name of the instance that you created.
+
+### Install the dependencies
+
+Update packages:
+
+```bash
+sudo apt-get update
+```
+
+Install git and npm:
+
+```bash
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
+```
+
+Install Node.js:
+
+```bash
+sudo apt-get install git nodejs
+```
+
+When asked if you want to continue, enter `Y`.
+
+### Install and run the frontend web app
+
+Clone the sample app:
+
+```bash
+git clone https://github.com/GoogleCloudPlatform/todomvc-mongodb.git
+```
+
+Install the app dependencies:
+
+```bash
+cd todomvc-mongodb; npm install
+```
+
+```bash
+sed -i -e 's/8080/80/g' server.js
+```
+
+Start the to-do web app:
+
+```bash
+sudo nohup nodejs server.js --be_ip [backend-internal-ip] --fe_ip [frontend-internal-ip] &
+```
+
+Replace `[frontend-internal-ip]` and `[frontend-internal-ip]` with 
+[internal IP addresses][spotlight-internal-ip] for the instances that you created.
```

Comment:
This spotlight item doesn't seem to be working. The external IP addresses item works fine.

Reply:
(Same as the other comment about this target.)